sdk/storage/azure-storage-file-share/swagger/src/main/java/com/azure/storage/file/share/customization/ShareStorageCustomization.java

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.share.customization;
+
+import com.azure.autorest.customization.ClassCustomization;
+import com.azure.autorest.customization.Customization;
+import com.azure.autorest.customization.LibraryCustomization;
+import com.azure.autorest.customization.MethodCustomization;
+import com.azure.autorest.customization.PackageCustomization;
+import com.azure.autorest.customization.PropertyCustomization;
+
+/**
+ * Customization class for File Share Storage.
+ */
+public class ShareStorageCustomization extends Customization {
```

Comment:
Is this something to do with the transforms?

Reply:
Yeah this is the new way to modify generated code